src/Parser/Core/Modules/Items/Legion/AntorusTheBurningThrone/EyeOfHounds.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import ItemLink from 'common/ItemLink';
+import Analyzer from 'Parser/Core/Analyzer';
+import Wrapper from 'common/Wrapper';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import { formatPercentage } from 'common/format';
+import { calculatePrimaryStat, calculateSecondaryStatDefault } from 'common/stats';
+import Abilities from 'Parser/Core/Modules/Abilities';
+
+/**
+ * Eye of Shatug / Eye of F'harg
+ * Use: swaps between both versions
+ * initial state is in the trinkets bonusID (3618 for the stamina one, 3619 for the armor one)
+*/
+
+const ARMOR_EYE_BONUS_ID = 3619;
```

Comment:
It's a bit confusing that you call this armor  and `isCurrentlyVers` vers when everything else uses stam/str in the names. Consider renaming one or the other so everything is vers/armor or stam/str.